server/models/development/update/synt.sql

```diff
@@ -217,3 +217,36 @@ ALTER TABLE `rubric`
 CHANGE `is_advance` `is_advance` tinyint(1) DEFAULT '0';
 ALTER TABLE `rubric`  
 CHANGE `is_social_care` `is_social_care` tinyint(1) DEFAULT '0';
+
+-- @sfount
+-- 2015-15-09
+-- Model languages
+-- UPDATE - Language table exists but did not propegate through application language settings
+
+/* DROP TABLE IF EXISTS `language`; */
+/* CREATE TABLE `language` ( */
+  /* `id` smallint(5) unsigned NOT NULL, */
+  /* `title` varchar(35) NOT NULL, */
+  /* `description` varchar(50), */
+  /* `translate_key` varchar(5) NOT NULL, */
+  /* `locale_key` varchar(5) NOT NULL */
+/* ) ENGINE=InnoDB DEFAULT CHARSET=latin1; */
+
+ALTER TABLE `language` CHANGE `key` `translate_key` varchar(5) NOT NULL;
+ALTER TABLE `language` ADD locale_key varchar(5);
+  
+UPDATE `language` SET locale_key = 'en-us' WHERE id = 1;
+UPDATE `language` SET locale_key = 'fr-cd' WHERE id = 2;
+UPDATE `language` SET locale_key = 'fr-cd' WHERE id = 3;
+
+ALTER TABLE `language` MODIFY `locale_key` varchar(5) NOT NULL;
+
+-- @sfount
+-- 2015-17-09
+-- Model enterprise (application) wide default language 
+ALTER TABLE `enterprise` ADD language_id tinyint(3);
```

Comment:
I remember we discussed adding languages to enterprises, but after seeing this code, would it make better sense to have languages associated with users?